### DISCUSSION TITLE
How to add data without running into `Maximum call stack size exceeded`

### DISCUSSION THREAD

**Discussion by DerAndereAndi** [apertureless/vue-chartjs#1044] [Q&A]
Hi everyone,

I am trying to add data to a line chart, which means adding an item to the labels and each datasets data property. But whenever I push an item to a datasets data property, I run into the error `Maximum call stack size exceeded`.

I also tried to get the demo https://www.chartjs.org/docs/latest/samples/line/line.html to work with this library, but to no luck so far. Does anyone have a hint on how to accomplish this?

Using ChartJS 4.3.0 and vue-chartjs 5.2.0.

Thanks

**Comment by apertureless** [comment]
Without seeing code or reproductions its hard to tell. Looks like you have an infinite loop in the reactive update.
